src/Atmos/Model/moisture.jl

```diff
@@ -57,6 +57,10 @@ vars_aux(::DryModel,FT) = @vars(θ_v::FT)
 end
 
 thermo_state(moist::DryModel, orientation::Orientation, state::Vars, aux::Vars) = PhaseDry(internal_energy(moist, orientation, state, aux), state.ρ)
+@inline function linearized_pressure(::DryModel, orientation::Orientation, state::Vars, aux::Vars)
```

Comment:
Wouldn't it be more logical to move this to `linear.jl` (along with the linearized pressure function from `MoistThermodynamics`? That way, all the linearized equations are together, rather than spread out over files.

Reply:
Done